tools/src/main/java/org/apache/kafka/tools/VerifiableConsumer.java

```diff
@@ -633,7 +632,8 @@ public static VerifiableConsumer createFromArgs(ArgumentParser parser, String[]
             brokerHostandPort = res.getString("brokerList");
         } else {
             parser.printHelp();
-            Exit.exit(0);
+            //Can't use `Exit.exit` here because `Exit` doesn't exists until 0.11.0.0.
+            System.exit(0);
```

Comment:
Can we submit the `Exit` changes as a separate PR? It needs to be cherry-picked to the 2.5 branch as well since it's causing system test failures.

Reply:
> Can we submit the Exit changes as a separate PR?

Yes, we can :)

Please, see - https://github.com/apache/kafka/pull/8197